model/trace/instrumentation/aws-sdk.yml

```diff
@@ -55,130 +54,21 @@ groups:
-                 "WriteCapacityUnits": number
-              },
-              "TableName": "string",
-              "WriteCapacityUnits": number
-            }'
-      - id: item_collection_metrics
-        type: string
-        stability: experimental
-        brief: "The JSON-serialized value of the `ItemCollectionMetrics` response field."
-        examples:
-          - '{
-              "string" : [
-                 {
-                    "ItemCollectionKey": {
-                       "string" : {
-                          "B": blob,
-                          "BOOL": boolean,
-                          "BS": [ blob ],
-                          "L": [
-                             "AttributeValue"
-                          ],
-                          "M": {
-                             "string" : "AttributeValue"
-                          },
-                          "N": "string",
-                          "NS": [ "string" ],
-                          "NULL": boolean,
-                          "S": "string",
-                          "SS": [ "string" ]
-                       }
-                    },
-                    "SizeEstimateRangeGB": [ number ]
-                 }
-              ]
-             }'
-      - id: provisioned_read_capacity
-        type: double
-        stability: experimental
-        brief: "The value of the `ProvisionedThroughput.ReadCapacityUnits` request parameter."
-        examples:
-          - 1.0
-          - 2.0
-      - id: provisioned_write_capacity
-        type: double
-        stability: experimental
-        brief: "The value of the `ProvisionedThroughput.WriteCapacityUnits` request parameter."
-        examples:
-          - 1.0
-          - 2.0
-      - id: consistent_read
-        type: boolean
-        stability: experimental
-        brief: "The value of the `ConsistentRead` request parameter."
-      - id: projection
-        type: string
-        stability: experimental
-        brief: "The value of the `ProjectionExpression` request parameter."
-        examples:
-          - Title
-          - Title, Price, Color
-          - Title, Description, RelatedItems, ProductReviews
-      - id: limit
-        type: int
-        stability: experimental
-        brief: "The value of the `Limit` request parameter."
-        examples:
-          - 10
-      - id: attributes_to_get
-        type: string[]
-        stability: experimental
-        brief: "The value of the `AttributesToGet` request parameter."
-        examples:
-          - lives
-          - id
-      - id: index_name
-        type: string
-        stability: experimental
-        brief: "The value of the `IndexName` request parameter."
-        examples:
-          - name_to_group
-      - id: select
-        type: string
-        stability: experimental
-        brief: "The value of the `Select` request parameter."
-        examples:
-          - ALL_ATTRIBUTES
-          - COUNT
+      - ref: aws.dynamodb.table_names
```

Comment:
@joaopgrassi One question for understanding.

In https://github.com/open-telemetry/semantic-conventions/blob/2541b872367cd4ca3119fd71ee2c468e09e4a3d5/model/general.yaml#L49 you added the requirement level explicitly (even for `recommended` which is the default, right?). Here we omit the requirement level, though it's also `recommended`. 

What do you think should be our recommendation? I think we should be consistent with that, either always setting it explicitly or omitting it when it's the default.

Reply:
I agree we might create a rule here as we do now with stability. 
I'm in favour of putting explicit requirement level everywhere. We could even enforce it through build-tools

Reply:
yeah it was probably a left over that I forgot to add. I agree with you folks, I prefer to have it explicitly, even though it's the default. I think this is a important enough property that I like having it there always. It's specially good for newcomers since they may not know the defaults.